.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ env:
 jobs:
   install-deps:
     name: Install and cache dependencies
-    if: ${{ !startsWith( github.event.pull_request.labels[0].name, 'autorelease' ) }}
+    if: ${{ !startsWith( github.ref_name, 'release-please--' ) }}
```

Comment:
Label is applied after the PR is open, so the previous logic did not work on the initial run.